### DISCUSSION TITLE
Overlaying text on an image

### DISCUSSION THREAD

**Discussion by iTheMaverick** [twbs/bootstrap#39437] [Q&A]
Somewhat new to Bootstrap, what's the easiest way to overlay this text overtop of the transparent image?
![image](https://github.com/twbs/bootstrap/assets/141296807/7784b0c2-dc35-49ab-bac6-370e407a2c73)

**Comment by iTheMaverick** [comment]
I was able to do it using:
```html
<section id="info" class="bg-primary py-5 px-5">
      <div class="container fluid text-center">
        <img src="img/ApexDesignsStudioWordsWebsiteLarge.png" class="pb-5 opacity-25 img-fluid" alt="ApexDesignsStudio">
        <p class="lead fs-2 fw-bold text-light carousel-caption" style="top: 104vh;">From custom logos and graphics to web design and search engine optimization (SEO), <br> our range of creative digital services will help you shine.</p>
      </div>
```
      But it's not responsive, what would be the best way to achieve this while keeping it responsive?
![image](https://github.com/twbs/bootstrap/assets/141296807/f436976a-5b4e-4aca-8141-03077f044428)


Examples of it moving around based on screen size:
![image](https://github.com/twbs/bootstrap/assets/141296807/8419ac9e-4e06-4248-8dbe-dce83e01d0b8)
![image](https://github.com/twbs/bootstrap/assets/141296807/383a9fb0-a91e-4349-bf19-46031ec61009)

**Comment by ascensus-mdb** [comment]
Try this:

https://mdbootstrap.com/docs/standard/content-styles/masks/

**Reply by iTheMaverick** [reply]
![image](https://github.com/twbs/bootstrap/assets/141296807/ffbc0612-e1a7-42ef-89ee-380a496c6451)

**Reply by ascensus-mdb** [reply]
You need to install MDBootstrap first instead of regular Bootstrap
https://mdbootstrap.com/docs/standard/getting-started/installation/

**Reply by iTheMaverick** [reply]
Is there a GitHub community for MDBootstrap? Followed the installation tutorial twice and it still isn't looking like it should, it's just standard HTML.

**Reply by iTheMaverick** [reply]
Disregard, got it working. Appreciate it.
